myokit/tests/data/formats/sbml/TwoComp.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If we're not using this one yet, I'd prefer it didn't come in until a future PR!

Reply:
Ok, I removed it.